src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.proto.System;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.HardwareConstants;
+import frc.robot.Constants.PneumaticsConstants.ElevatorConstants;
+import frc.robot.util.Time;
+
+public class Elevator {
+    
+// double solenoid to control the pistons */
+    private DoubleSolenoid doubleSolenoid =
+      new DoubleSolenoid(
+          HardwareConstants.REV_PH_ID,
+          PneumaticsModuleType.REVPH,
+          ElevatorConstants.IN,
+          ElevatorConstants.OUT);
+
+ /** Create new pnuematic system */
+    public elevator() {
```

Comment:
Rename constructor from `elevator()` to `Elevator()`. Should fix the build issues.